server/const.go

```diff
@@ -1,9 +1,18 @@
 package main
 
 import (
+	"fmt"
+
+	"github.com/pkg/errors"
 	"github.com/prometheus/prometheus/model/labels"
 )
 
+const (
+	scrapeAcceptHeader = `application/openmetrics-text;version=1.0.0,application/openmetrics-text;version=0.0.1;q=0.75,text/plain;version=0.0.4;q=0.5,*/*;q=0.1`
+)
+
 var (
-	sampleMutator func(labels.Labels) labels.Labels //nolint:unused
+	sampleMutator    func(labels.Labels) labels.Labels //nolint:unused
+	errBodySizeLimit = errors.New("body size limit exceeded")
+	UserAgent        = fmt.Sprintf("%s/%s", PluginName, ScraperVersion)
```

Comment:
Can we change this to a constant?

Also, it looks slightly weird to see `var`s in a file named const.go :stuck_out_tongue:

Reply:
Haha, sure I'll re-organize no worries.